package.json

```diff
@@ -76,6 +76,7 @@
     "astro-benchmark": "workspace:*"
   },
   "devDependencies": {
+    "@babel/plugin-transform-react-jsx": "^7.22.5",
```

Comment:
This was necessary because the Preact Vite plugin provides this to babel as a string. So babel must resolve it and doesn't know where to do so.

Reply:
Maybe @bluwy knows a workaround

Reply:
I don't think this will work in user's projects? They'd have to install it themselves too to workaround it like here.

One thing I did try in the past (which has a catch) is setting the babel [cwd option](https://babeljs.io/docs/options#cwd) to the preact plugin [babel options](https://github.com/preactjs/preset-vite/blob/3ed6fb180ed3845f26281675e45f06c1acbce906/src/index.ts#L129). Assigning it to the resolved path of `@preact/preset-vite` could work I think, so the plugins are resolved from that directory. The catch is that if we ever pass additional babel plugins, it won't work.

Reply:
I tried installing it into the test project and that didn't work. I guess I don't understand the problem properly because it feels like this would always be breaking for every user of this plugin and they would have fixed it by now, or there at least would be an issue with a lot of activity on it. But there's not, so that's why I thought it was related to our monorepo setup.

Reply:
This is fixed via `cwd` config.